components/page-templates/B2B/CreateNewQuoteTemplate/CreateNewQuoteTemplate.tsx

```diff
@@ -0,0 +1,267 @@
+              <InputLabel shrink={true} sx={{ position: 'relative' }}>
+                {t('expiration-date')}
+              </InputLabel>
+              <Typography>{expirationDate}</Typography>
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <Typography variant="h2" mb={2}>
+            {t('quote-summary')}
+          </Typography>
+          <B2BProductSearch onAddProduct={handleAddProduct} />
+        </Grid>
+        <Grid item xs={12}>
+          <Stack gap={3}>
+            {mdScreen ? (
+              <B2BProductDetailsTable
+                items={quoteItems as CrOrderItem[]}
+                fulfillmentLocations={fulfillmentLocations}
+                purchaseLocation={purchaseLocation}
+                onFulfillmentOptionChange={onFulfillmentOptionChange}
+                onQuantityUpdate={handleQuantityUpdate}
+                onStoreSetOrUpdate={handleProductPickupLocation}
+                onItemDelete={handleDeleteItem}
+              />
+            ) : (
+              <Stack spacing={2}>
+                {quoteItems && quoteItems?.length > 0 ? (
+                  <CartItemList
+                    cartItems={quoteItems as CrOrderItem[]}
+                    fulfillmentLocations={fulfillmentLocations as Location[]}
+                    purchaseLocation={purchaseLocation}
+                    onCartItemDelete={handleDeleteItem}
+                    onCartItemQuantityUpdate={handleQuantityUpdate}
+                    onFulfillmentOptionChange={onFulfillmentOptionChange}
+                    onProductPickupLocation={handleProductPickupLocation}
+                    onCartItemActionSelection={() => null}
+                  />
+                ) : (
+                  <Typography variant="body1" sx={createNewQuoteTemplateStyles.noCartItems}>
+                    {t('search-to-add-products')}
+                  </Typography>
+                )}
+              </Stack>
+            )}
+
+            {!mdScreen && quoteItems?.length ? (
+              <Box paddingY={1} display="flex" flexDirection={'column'} gap={2}>
+                <LoadingButton variant="contained" color="primary" fullWidth>
+                  {t('submit-for-approval')}
+                </LoadingButton>
+                <Box display="flex" gap={3}>
+                  <LoadingButton
+                    variant="contained"
+                    color="secondary"
+                    fullWidth
+                    sx={{ padding: '0.375rem 0.5rem' }}
+                  >
+                    {t('clear-changes')}
+                  </LoadingButton>
+                  <LoadingButton variant="contained" color="inherit" fullWidth>
+                    {t('save-and-exit')}
+                  </LoadingButton>
+                </Box>
+              </Box>
+            ) : null}
+          </Stack>
+        </Grid>
+      </Grid>
+    </>
+  )
+}
+
+export default CreateNewQuoteTemplate
```

Comment:
Vertical Spacing and Fonts are not matching
![image](https://github.com/KiboSoftware/nextjs-storefront-b2b/assets/89385472/04d5ac06-6094-4c4b-bad8-237eab26ce56)

Reply:
Resolved